apps/web/src/routes/app/account/+page.svelte

```diff
@@ -131,34 +135,82 @@
 		</Tabs.Content>
 		<Tabs.Content value="billing">
 			<Card.Root class="overflow-hidden rounded-lg shadow-xl">
-				<Card.Header class="text-on-primary bg-gradient-to-r p-6">
-					<h2 class="text-xl font-semibold">Membership</h2>
+				<Card.Header class="p-6">
+					<h2 class="text-2xl font-bold">Your Subscription</h2>
 				</Card.Header>
 				<Card.Content class="p-6">
 					<div class="space-y-8">
-						<div class="bg-background space-y-4 rounded-lg bg-gradient-to-r p-6 shadow-sm">
-							<div class="text-2xl font-semibold">Current Plan</div>
-							<p class="font-medium">Starter</p>
-							<div class="mt-4 flex gap-4 transition-all duration-200 ease-in-out">
-								<Button class=" border bg-transparent text-current " href="/app/subscription"
-									>Change Plan</Button
-								>
-								<Button
-									class=" flex items-center gap-2 rounded-md border bg-transparent  text-current"
-								>
-									<Plus /> Add a promo code
-								</Button>
+						<div class="from-background-950 to-primary-800 bg-background rounded-lg">
+							<div class="rounded-lg p-6 pt-2">
+								<div class="mb-4 flex items-center justify-between">
+									<div>
+										<h4 class="text-lg font-semibold">{$subscriptionStore.tier?.name}</h4>
+										<p>{$subscriptionStore.sub.plan.interval}ly Subscription</p>
```

Comment:
When the user haven’t subscribed yet `$subscriptionStore.sub` is `null`. So check for that and when it's null maybe put the PricingTiers component there. Otherwise the app crashes when navigating to /account.